DMX.Sdk/Models/Clients/DmxEnvironments.cs

```diff
@@ -0,0 +1,13 @@
+﻿// --------------------------------------------------------------- 
+// Copyright (c) Microsoft Corporation. All rights reserved. 
+// ---------------------------------------------------------------
+
+
+namespace DMX.Sdk.Models.Clients
+{
+    public struct DmxEnvironments
+    {
+        public static string DEVELOPMENT = "https://dmx.dev";
```

Comment:
this should be `https://dmx-gatekeeper-dev.azurewebsites.net`
same thing for production - replace `dev` with `prod`